dev/integration_tests/ios_platform_view_tests/ios/PlatformViewUITests/PlatformViewUITests.m

```diff
@@ -36,17 +36,30 @@ - (void)setUp {
   if ([appIcon waitForExistenceWithTimeout:kStandardTimeOut]) {
     NSLog(@"Deleting previously installed app.");
 
-    // Make icons wiggle
-    [appIcon pressForDuration:3];
+    // It's possible that app icon is not hittable yet.
+    NSPredicate *hittable = [NSPredicate predicateWithFormat:@"exists == YES AND hittable == YES"];
+    [self expectationForPredicate:hittable evaluatedWithObject:appIcon handler:nil];
+    [self waitForExpectationsWithTimeout:kStandardTimeOut handler:nil];
+
+    // Pressing for 2 seconds will bring up context menu.
+    // Pressing for 3 seconds will dismiss the context menu and make icons wiggle.
+    [appIcon pressForDuration:2];
```

Comment:
I hope this isn't flaky...  You could consider taking `XCAttachment` screenshots when these fail to see what's going on.
I hate these kinds of Xcode test harness bugs.  As do you too now!

Reply:
Yeah I was thinking the same. In that stackoverflow I linked, there are 2 versions: 1.3 seconds and 2 seconds. 

I picked 2 seconds in case of slow hardware.

Reply:
Added screenshot

Reply:
Hm actually sorry to thrash on this, but never mind this isn't necessary, the next thing that happens is the `screenshotAttachment` `XCTAssert` that will automatically take a screenshot if the assert fails.